### PR TITLE
22.3 fb finance public fix

### PR DESCRIPTION
**Pull request by jonesgaohsu**
#### Rationale
Update of Invoice Items for Publix, the webpart of  public finance and Linked Schema Templates

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

**Review by kollil — APPROVED** [review]
looks good

File: onprc_billing/resources/queries/onprc_billing/publicInvoicedItems.sql [hunk 1]
```diff
@@ -12,51 +12,58 @@
...
+    i.mailcode,
+    i.contactPhone,
+    i.chargeId,
+    i.objectid,
+    i.quantity,
+    i.unitCost,
+    i.totalcost,
+    i.chargeCategory,
+    pr.investigatorid as Inv1,
+    pr.project as pr1
```
Comment (labkey-martyp) [inline onprc_billing/resources/queries/onprc_billing/publicInvoicedItems.sql:48]: You can remove these fields if you want.  This was just part of our troubleshooting.

**Review by labkey-martyp — COMMENTED** [review]
This all looks good Gary. Aligns with our meeting yesterday.

**Review by labkey-jeckels — APPROVED** [review]
I concur with Marty - these changes look good. I've gotten all queries to validate locally by defining the required linked schemas.

Followup activities, which can all happen post-upgrade include:

- Consolidate the duplicate linked schemas that Gary and Lakshmi introduced in different FBs. Queries reference both variants at the moment. At a minimum, we should be able to consolidate `financepublic` and `publicfinance`
- Delete `RandalData_MHCData.sql` which will be broken on the production server.
- Update the automated tests so that they can continue to provide useful validation of all of the queries in the modules.

**Comment by jonesgaohsu** [comment]
I added the publicInvoicedItems.sql in the FB 22.3_fb_FinanceUpdate, and plan to remove it from this FB after cleaning up teh code suggestions from Marty.  So glad to have multiple eyes on our process.

**Review by kollil — APPROVED** [review]
looks good

**Comment by kollil** [comment]
> I concur with Marty - these changes look good. I've gotten all queries to validate locally by defining the required linked schemas.
> 
> Followup activities, which can all happen post-upgrade include:
> 
> * Consolidate the duplicate linked schemas that Gary and Lakshmi introduced in different FBs. Queries reference both variants at the moment. At a minimum, we should be able to consolidate `financepublic` and `publicfinance`
> * Delete `RandalData_MHCData.sql` which will be broken on the production server.
> * Update the automated tests so that they can continue to provide useful validation of all of the queries in the modules.

@labkey-jeckels, Yes, the plan is to consolidate the duplicate templates `financepublic` and `publicfinance`, it on my list to do after the upgrade. Thank you - Lakshmi

**Comment by jonesgaohsu** [comment]
This will be replaced with more complete code

**Pull request closed without merging**